docs/security/faq.md

```diff
@@ -95,7 +95,7 @@ Such is the case of recent `sudo` vulnerability.
 
 This project mitigates these issues by:
 
-* Automating creation of filesystem items and setting proper permissions by default. This removes the possibility of mistakes.
+* Automating creation of file-system items and setting proper permissions by default. This removes the possibility of mistakes.
```

Comment:
"filesystem" is term usually used by programmers. using hyphen doesn't feel right. I believe I've seen it with space somewhere.

Reply:
```suggestion
* Automating creation of filesystem items and setting proper permissions by default. This removes the possibility of mistakes.
```